pages/dkp/konvoy/2.1/choose_infrastructure/aws/advanced/delete/index.md

```diff
@@ -138,4 +138,4 @@ INFO[2021-06-09T12:15:20-07:00] Deleting bootstrap cluster                    sr
 
 - The Konvoy version used to create the workload cluster must match the Konvoy version used to delete the workload cluster.
 
-[makeselfmanaging]: ../self-managing
```

Comment:
@dlipovetsky suggested adding a redirect here to the new page.
Can someone please point me to where the redirects are setup in this repo.

AFAIK its an nginx config somewhere in this repo.

Reply:
Maybe here https://github.com/mesosphere/dcos-docs-site/blob/main/s3config.json?

Reply:
Actually this only makes sense when we make `latest/` redirect to `2.1`.

Created a branch https://github.com/mesosphere/dcos-docs-site/commit/ab9f022e8acf48f702fd4a511704b7392026e85f that we can commit when that change is made.

Reply:
Yeah the redirect will happen at GA.